src/main/java/group1/intern/repository/customization/OrderCustomRepository.java

```diff
@@ -0,0 +1,13 @@
+package group1.intern.repository.customization;
+
+import group1.intern.model.Enum.OrderStatus;
+import group1.intern.model.Order;
+import group1.intern.repository.base.BaseRepository;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+
+public interface OrderCustomRepository extends CustomRepository<Order, Integer, BaseRepository<Order>> {
```

Comment:
```suggestion
public interface OrdersRepository extends CustomRepository<Order, Integer, BaseRepository<Order>> {
```
thêm chữ custome chi thế cu, chị thấy đây là repository trỏ đến Order class nè